src/win/pty.cc

```diff
@@ -141,7 +141,7 @@ static NAN_METHOD(PtyGetExitCode) {
     return;
   }
 
-  uint32_t pidHandle = info[0]->Uint32Value();
+  int32_t pidHandle = info[0]->Uint32Value();
   DWORD exitCode = 0;
   GetExitCodeProcess((HANDLE)pidHandle, &exitCode);
```

Comment:
@Tyriar Minor optimization idea: imho you can even avoid the stack push/pop by giving the signed value back in EAX:
```C++
DWORD exitCode = 0;
GetExitCodeProcess((HANDLE) info[0]->IntegerValue(), &exitCode);
```
Well, doesnt really matter at this point ;)